domains/client/domain-executor/src/bundle_election_solver.rs

```diff
@@ -192,3 +196,13 @@ where
         Ok(None)
     }
 }
+
+/// Computes the VRF signature hash.
+fn vrf_signature_hash(vrf_signature: &VRFSignature) -> Blake2b256Hash {
+    // TODO: the upstream version of VRFSignature has Encode trait,
```

Comment:
Could you specify in the TODO which commit upstream you mean so it's easier to verify

Reply:
No longer relevant